apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx

```diff
@@ -66,98 +63,67 @@ export const UsePoolerCheckbox = ({
-                                            </Tooltip.Content>
-                                          </Tooltip.Portal>
-                                        )}
-                                      </Tooltip.Root>
-                                    </CommandItem_Shadcn_>
-                                    <CommandItem_Shadcn_
-                                      value="session"
-                                      className="cursor-pointer w-full text-foreground"
-                                      onSelect={() => {
-                                        onSelectPoolingMode('session')
-                                        setOpen(false)
-                                      }}
-                                      onClick={() => {
-                                        onSelectPoolingMode('session')
-                                        setOpen(false)
-                                      }}
-                                    >
-                                      Session mode
-                                    </CommandItem_Shadcn_>
-                                  </CommandGroup_Shadcn_>
-                                </CommandList_Shadcn_>
-                              </Command_Shadcn_>
-                              <Separator />
-                              <p
-                                className="px-3 py-2 text-xs text-foreground-light hover:text-foreground cursor-pointer transition"
+                          <DropdownMenu>
+                            <DropdownMenuTrigger asChild>
+                              <Button
+                                type="outline"
+                                className="py-0.5 pr-1.5"
+                                iconRight={<IconChevronDown strokeWidth={1.5} />}
+                              >
+                                <span className="capitalize">Mode: {poolingMode}</span>
+                              </Button>
+                            </DropdownMenuTrigger>
+                            <DropdownMenuContent side="bottom" align="start">
+                              <DropdownMenuItem
+                                key="transaction"
+                                className="text-foreground"
+                                onClick={() => onSelectPoolingMode('transaction')}
+                              >
+                                <Tooltip.Root delayDuration={0}>
+                                  <Tooltip.Trigger asChild>
+                                    <span className="w-full">Transaction mode</span>
+                                  </Tooltip.Trigger>
+                                  {data.pool_mode === 'session' && (
+                                    <Tooltip.Portal>
+                                      <Tooltip.Content side="right">
+                                        <Tooltip.Arrow className="radix-tooltip-arrow" />
+                                        <div
```

Comment:
this div is random, shouldn't all these classes be in the `Tooltip.Content` ?

Reply:
this isn't a UI library component so its unstyled - although yes we should move it to the UI lib as you mentioned in the other comment

Reply:
yea but tooltip.content takes css classes. don't need to add more divs in it

Reply:
if you've done this for every other one it doesn't matter

Reply:
^ ahh yeah - been doing this for every other one, no worries will be resolved when we move all to shadcn's tooltips
i'll do that next week